script/Ciel/Day 10/10_01_CIEL10_6.txt

```diff
@@ -1297,47 +1434,52 @@ Releasing it on me, on the third person dazedly standing around, such was the wo
 [sha:7b956eb6a50f632dcfedaed309eb4f2c4f2395ad]{
 -- Page 507, Offset 28496.
 -- ……だが、その秘策は実行すらされなかった。
-However... That secret plan could not even be set into motion.
+Except, she doesn't even get a chance to carry out the second part of her plan.
 }
 [sha:aba7f14c6a9b1ac1c865b8836b70c458f1fac491]{
 -- Page 507, Offset 28497.
 -- <女|ノエル>が黒鍵三本を放った時点で、その右肩にはシエル先輩の黒鍵が突き刺さっていたのだから。
-Indeed, the moment Noel throws her three Black Keys, her right shoulder gets transpierced by Ciel-senpai's Black Key.
+The instant those three Black Keys left her hand,
+ her right shoulder was skewered by one of Ciel-senpai's Black Keys.
 }
 [sha:e78fa8c2536d2b576940742469cc4a4251d7ebbf]{
 -- Page 508, Offset 28498.
 -- 「きゃあああああああああああーーーー！！！」
-"Arghhhhhhhhhhhhhhh――――!!!!"
+"%{i}Arghhhhhhhhhhhhhhh%{/i}―――!!!!"
 }
 [sha:6d8081d7eadf8088104d1baefdd3272cd32b88f3]{
 -- Page 508, Offset 28499.
 -- 耳をつんざく閃光と悲鳴。
-A flash of light and a ear-splitting scream of anguish.
+There's a flash of light,
+ followed by ear-splitting screams of anguish.
 }
 [sha:404ff12a0d46d8e07959f80367c37da53edfcd27]{
 -- Page 508, Offset 28500.
 -- <女|ノエル>の右肩に刺さった黒鍵が放電し、全身の筋肉を舐めるように感電させる。
-The Black Key lodged in the woman's right shoulder discharges electricity, making all the muscles of her body feel like they are on fire.
+The Black Key lodged in the woman's right shoulder unleashes a jolt of electricity that rips through all the tissues in her body.
```

Comment:
tissue should be singular.

Reply:
You sure? There's at least like 4 tissues in the body

Reply:
tissues is correct

Reply:
Google says both are correct, but singular is more common.

Reply:
I still don't trust a raw google search for this kind of thing. There are too many confounding elements. An actual text search shows matches for plural, but not for singular:
https://ludwig.guru/s/all+the+tissues
https://ludwig.guru/s/all+the+tissue

ngrams _also_ prefers plural:
![2023-01-06-202502_1788x637_scrot](https://user-images.githubusercontent.com/2476799/211124907-87c448e3-6a62-4048-809b-3e4b677558bc.png)
![2023-01-06-202219_1769x682_scrot](https://user-images.githubusercontent.com/2476799/211124908-99a3df4b-e379-48f7-bf5e-40c35df007db.png)

Ben and I also both seem to think plural sounds correct, so inclined to go with that.